packages/@okta/i18n/src/properties/login.properties

```diff
@@ -1031,7 +1031,7 @@ oie.enroll.okta_verify.setup.title = Set up Okta Verify
 oie.enroll.okta_verify.setup.email.title = Check your email
 oie.enroll.okta_verify.setup.sms.title = Check your text messages
 oie.enroll.okta_verify.setup.skipAuth.subtitle = To set up Okta Verify on additional devices, you can copy an existing Okta Verify account onto a new device.
-oie.enroll.okta_verify.setup.skipAuth.openOv = Open Okta Verify on any of your other Okta Verify devices (<$1>{0}</$1>).
+oie.enroll.okta_verify.setup.skipAuth.openOv = Open Okta Verify on any of your other Okta Verify devices (Such as your <$1>{0}</$1>).
```

Comment:
I don't think we can make this change because this string is already translated for self service EA. We will have to change for phase 2 @stevennguyen-okta

Reply:
@helenchen-okta i understand but since this is just a small one, i'm fine with adding it and get it in next month's translation. we will need it eventually and this doesn't seems to be a blocker to me

Reply:
Similar with line 1019-1020, please add a brief comment for the arguments.